### PR TITLE
feat(dashboards): Widget Viewer more relevant table columns

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Updates Line, Area, Big Number, and Bar charts in Widget Viewer to display a fixed set of columns (title, event.type, project, etc)
This is an improvement over the previous table which just displayed the yAxis valid from the chart which wasn't very helpful.
image

### CODE REVIEW

**Review by narsaynorath — APPROVED** [review]

File: static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx [hunk 1]
```diff
@@ -159,6 +165,56 @@ export const renderGridBodyCell =
...
+
 const StyledTooltip = styled(Tooltip)`
   display: initial;
 `;
+
+const PrependHeader = styled('span')`
+  color: ${p => p.theme.subText};
+`;
+
+const StyledLink = styled(Link)`
```
Comment (narsaynorath) [inline static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx:216]: What happens if we just use `Link`? I removed this and used `Link` and it looks fine for me

**Review by narsaynorath — APPROVED** [review]
Just a small comment this time

File: static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx:216]: good question Nar! can value be also an element of just a text/number/boolean? Unfortunately atm it is of type any

File: static/app/components/modals/widgetViewerModal.tsx [hunk 2]
```diff
@@ -88,47 +92,67 @@ function WidgetViewerModal(props: Props) {
...
-      modalSelection,
-      tableWidget.displayType
+  // Default table columns for visualizations that don't have a column setting
+  const shouldReplaceTableColumns = [
+    DisplayType.AREA,
+    DisplayType.LINE,
+    DisplayType.BIG_NUMBER,
+    DisplayType.BAR,
+  ].includes(widget.displayType);
+  if (shouldReplaceTableColumns) {
```
Comment (priscilawebdev) [inline static/app/components/modals/widgetViewerModal.tsx:119]:
```suggestion
  ].includes(widget.displayType);
  
  if (shouldReplaceTableColumns) {
```

File: static/app/components/modals/widgetViewerModal.tsx [hunk 3]
```diff
@@ -88,47 +92,67 @@ function WidgetViewerModal(props: Props) {
...
+  );
+  const columnOrder = eventView.getColumns();
+  const columnSortBy = eventView.getSorts();
+
+  const queryOptions = sortedQueries.map(({name, conditions}, index) => ({
+    label: truncate(name || conditions, 120),
+    value: index,
+  }));
 
+  const renderWidgetViewer = () => {
```
Comment (priscilawebdev) [inline static/app/components/modals/widgetViewerModal.tsx:155]:
```suggestion
  function renderWidgetViewer() {
```

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**